flexmeasures/data/models/planning/utils.py

```diff
@@ -109,3 +110,15 @@ def get_prices(
                 "Prices partially unknown for planning window."
             )
     return price_df, query_window
+
+
+def check_required_attributes(sensor: Sensor, attributes: List[str]):
```

Comment:
Some suggestions on this  function:

- How about checking the attribute type as well? We would only do an `isinstance` check here when we find the attribute. If we wanted, we could try to parse strings to the required type.

- This function also can go to a higher-level util module, as I believe it'll be useful in other contexts. Or move it to the `Sensor` class?


As a result, this would implement something like the JSONSchema I suggested in PR #242, but in a light way (each context can check the attributes it requires)